### PR TITLE
add cyrillic ё and й

### PR DESCRIPTION
**Pull request by glebtv**

### CODE REVIEW

**Comment by maxcnunes** [comment]
@tyxla could you please merge and release this one? 🙏

**Comment by tyxla** [comment]
Sure thing @maxcnunes, I'm planning some work on the library in the following days. Thanks for the PR @glebtv!

File: test.js [hunk 1]
```diff
@@ -11,8 +11,18 @@ tape('remove accents from string', function(t) {
...
+	var expected = 'ЕеИи';
+
+	t.same( output, expected );
+
+	t.end();
+});
+
 tape('do not modify non-accented strings', function(t) {
-	var input = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz123456789.,:;~`!@#$%^&*()-_=+[]{}\'"|\\<>?/';
+	var input = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz123456789.,:;~`!@#$%^&*()-_=+[]{}\'"|\\<>?/лЛиИ';
```
Comment (tyxla) [inline test.js:25]: Hey, @glebtv this appears to declare `лЛ` while I would have assumed it declares `Ее` instead. Did you mean to use `Ее`?

Comment (glebtv) [reply test.js]: Changed to test e E and и И instead

**Comment by glebtv** [comment]
Hello, changed the test and updated the branch. Please merge,

File: test.js [hunk 2]
```diff
@@ -11,8 +11,18 @@ tape('remove accents from string', function(t) {
 	t.end();
 });
 
+tape('remove cyrillic accents from string', function(t) {
+	var input = 'ЁёЙй';
```
Comment (tyxla) [inline test.js:15]: Ideally, this would go into the large test that covers all supported characters, but not a big deal.

File: package.json [hunk 3]
```diff
@@ -3,9 +3,7 @@
   "version": "0.4.3",
   "description": "Converting the accented characters to their corresponding non-accented ASCII characters.",
   "main": "index.js",
-  "dependencies": {
-
-  },
+  "dependencies": {},
```
Comment (tyxla) [inline package.json:6]: Unrelated change, but I'm happy to take it 👍

**Review by tyxla — APPROVED** [review]
Thanks a bunch, @glebtv!

**Pull request merged**